src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -0,0 +1,415 @@
+                <div class="details-icon" @click=${this._emailUser}>${emailSVG}${email}</div>
+                <div class="details-icon" @click=${this._callUser}>${phoneSVG}${phone}</div>
+                <div class="details-icon">${locationSVG}${location}</div>
+              </div>
+              <div class="section-divider"></div>
+              <div class="custom-section">
+                ${this.renderTemplate('additional-details', null)}
+              </div>
+            </div>
+          </div>
+        </div>
+      `;
+    } else {
+      return html`
+        <div class="additional-details-button">
+          <svg
+            class="additional-details-svg"
+            width="16"
+            height="15"
+            viewBox="0 0 16 15"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path d="M15 7L8.24324 13.7568L1.24324 6.75676" stroke="#3078CD" />
+          </svg>
+        </div>
+      `;
+    }
+  }
+
+  private _showAdditionalDetails(e: Event) {
+    // handles clicking when parent is also activated by click
+    e.stopPropagation();
+    const root = this.renderRoot.querySelector('.root');
+    if (root && root.animate) {
+      // play back
+      root.animate(
+        [
+          {
+            transformOrigin: 'top left',
+            height: 'auto'
+          },
+          {
+            transformOrigin: 'top left',
+            height: 'auto'
+          }
+        ],
+        {
+          duration: 1000,
+          easing: 'ease-in-out',
+          fill: 'both'
+        }
+      );
+
+      this.showadditionaldetails = 'true';
+    }
+  }
+
+  private _callUser(e: Event) {
+    const user = this.personDetails;
+    // tslint:disable-next-line: one-variable-per-declaration
+    let phone;
+
+    if ((user as MicrosoftGraph.User).businessPhones && (user as MicrosoftGraph.User).businessPhones.length > 0) {
+      phone = (user as MicrosoftGraph.User).businessPhones[0];
+    }
+    e.stopPropagation();
+    window.location.assign('tel:' + phone);
+  }
+
+  private _emailUser(e: Event) {
+    const user = this.personDetails;
+    let email;
+
+    if ((user as MicrosoftGraph.User).mail) {
+      email = (user as MicrosoftGraph.User).mail;
+    }
+    e.stopPropagation();
+    window.location.assign('mailto:' + email);
+  }
+
+  private _chatUser(e: Event) {
+    const user = this.personDetails;
+    // tslint:disable-next-line: one-variable-per-declaration
+    let chat;
+
+    if ((user as MicrosoftGraph.User).mailNickname) {
+      chat = (user as MicrosoftGraph.User).mailNickname;
+    }
+    e.stopPropagation();
+    window.location.assign('sip:' + chat);
+  }
+
+  private async loadData() {
+    if (this.personDetails) {
+      return;
+    }
+
+    const provider = Providers.globalProvider;
+
+    if (!provider || provider.state !== ProviderState.SignedIn) {
+      return;
+    }
+
+    // TODO: load data from the graph
```

Comment:
Can you move this into an issue on GitHub and remove the comment?

Reply:
4306983